### PR TITLE
Master

### PR DESCRIPTION
**Pull request by josesho**
update to v 0.325.1

### CODE REVIEW

**Comment by josesho** [comment]
update to v0.325.1, add build noarch python

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  noarch: python
```
Comment (jakirkham) [inline recipe/meta.yaml:15]: This requires a [re-rendering]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) to fix up the feedstock (e.g. cutdown on CIs and update the README). Could you please submit one in a new PR?

Comment (josesho) [reply recipe/meta.yaml:15]: Hi, not exactly sure what change is requested? Also this is an older version of my package, not so sure if I want to push it up?

Comment (jakirkham) [reply recipe/meta.yaml:15]: Am requesting that the feedstock be re-rendered.

Comment (jakirkham) [reply recipe/meta.yaml:15]: Should add have only commented on this diff to draw attention to the fact that [adding `noarch: python` requires a re-rendering]( https://conda-forge.org/docs/meta.html#building-noarch-packages ). Though the re-rendering can and should be made using the latest commit of this recipe. Please let me know if there are other questions.

Comment (josesho) [reply recipe/meta.yaml:15]: ok, noted, have re-rendered on the latest release PR #12 .

Comment (jakirkham) [reply recipe/meta.yaml:15]: Thanks.

**Pull request merged**